doc/UsersGuide/source/solving.rst

```diff
@@ -327,6 +327,41 @@ system.
 
   body2.frame_a.r_0[1]
 
+Importing initial values from previous simulations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In many use cases it is useful to import initial values from previous simulations, possibly obtained with
+another Modelica tool, and saved in a .mat file. There are two different options to do that.
+
+The first option is to solve the initial equations specified by the Modelica model, using the previous simulation results to
+obtain good initial guesses for the iterative solvers. This can be very helpful in case the initialization problem involves the
+solution of large nonlinear systems of equations by means of iterative algorithms, whose convergence is sensitive to the selected
+initial guess. Importing a previously found solution allows the OpenModelica solver to pick very good initial guesses for the
+unknowns of the iterative solvers, thus achieving convergence with a few iterations at most. Since the initial equations
+are solved anyway, the values of all variables and derivatives, as well as of all parameters with `fixed = false` attribute,
+are re-computed and fully consistent with the selected initial conditions, even in case the previously saved simulation results
+refer to a slightly different model configuration. Note that parameters with `fixed = true` will also get their values from the
+imported .mat file, so if you want to change them you need to edit the .mat file accordingly. 
```

Comment:
Maybe move the last sentence further down. I wasn't sure what the imported .mat file is, when reading this the first time.

Reply:
It is mentioned in the first sentence of the section "In many use cases it is useful to import initial values from previous simulations, possibly obtained with another Modelica tool, and saved in a .mat file. There are two different options to do that."

I changed it into "In many use cases it is useful to import initial values from previous simulations, possibly obtained with
another Modelica tool, **which are** saved in a .mat file. There are two different options to do that."